### DISCUSSION TITLE
Denon AVR-1513 Arduino Remote Control

### DISCUSSION THREAD

**Discussion by 79robb** [Arduino-IRremote/Arduino-IRremote#1277] [Show and tell]
Hi,

I'll start by saying this library is awesome, thanks to all who contribute and keep it going.

I have created an abomination of an IR controller consisting of an Arduino ESP32 and an IR LED I found in a scrap remote. I tied it all together with the Arduino Cloud and made a mobile dashboard with some buttons (Volume control, power, source selection etc) to control the basic functions of the device.

All this because I bought a second hand Denon AVR-1513 a while ago but received no remote. I have succeeded in my goal of not having to get off the sofa to change the volume, that works perfectlyish. However the setup menu, navigation particularly, is proving more challenging.

I can open the setup menu, I can press enter in the menu and I can use the left and right cursor commands. I can't find a return command in the documentation I've been using, link at the end, but that's not important as I can press the setup button to close the menu and open it again, returning me to the highest level in the menu.

The only problem I'm having is the up and down commands. I can press up **or** down once. After that up and down do nothing. If I close and re open the setup menu I can again press up **or** down once.

In summary I'm pretty happy with what I've managed so far. I'm not explicitly looking for help, just on the off chance someone knows what's happening here with the up and down. 

Again this library is so cool, can't wait to see what else I can use it for. Thanks <3

https://assets.denon.com/documentmaster/uk/avr1513_ir_code_v01.pdf

**Comment by ArminJo** [comment]
Hi, I recently updated the Denon sources in the repository, but did not release them yet.
Make sure you use the new sources in your project 😀 .
